src/styles/styles.css

```diff
@@ -22,38 +22,37 @@
   }
 }
 
-::page-transition-outgoing-image(root) {
+::view-transition-old(root) {
   animation: 90ms cubic-bezier(0.4, 0, 1, 1) both fade-out,
     300ms cubic-bezier(0.4, 0, 0.2, 1) both slide-to-left;
 }
 
-::page-transition-incoming-image(root) {
+::view-transition-new(root) {
   animation: 210ms cubic-bezier(0, 0, 0.2, 1) 90ms both fade-in,
     300ms cubic-bezier(0.4, 0, 0.2, 1) both slide-from-right;
 }
 
-::page-transition-outgoing-image(movie-poster),
-::page-transition-incoming-image(movie-poster) {
+::view-transition-pair(movie-poster) {
```

Comment:
```suggestion
::view-transition-old(movie-poster),
::view-transition-new(movie-poster) {
```

Reply:
Thanks for the fix, but, for my understanding, I thought `pair` was basically for the use case of applying to both `old`+`new` ?